### DISCUSSION TITLE
About sliding_window_inference

### DISCUSSION THREAD

**Discussion by uto-lt** [Project-MONAI/MONAI#6536] [General]
Hi, @a-parida12
Could you please show your sample code about using this function in multiple inputs and single output scenarios

**Comment by a-parida12** [comment]
My model `forward` method required 2 inputs. I stacked my `2` inputs in the channel dimension and passed it thru a function called `bunched_infer` which called the actual forward call. See the example below-

```python 
def bunched_infer(inp):
        out = model.forward(inp[:,inp.shape[1]//2:,...], inp[:, :inp.shape[1]//2,...])
        return out
 
 inp1 = data1['image']
 inp2 = data2['image']
 stack_inp = torch.cat([inp1, inp2], axis = 1) 
 
 pred1= sliding_window_inference(stack_inp, roi_size=(16, 16, 16), sw_batch_size=1, predictor=bunched_infer, overlap=0.50, mode='gaussian')     
``` 

This answer works with the original query I had https://github.com/Project-MONAI/MONAI/discussions/4318#discussioncomment-4199756

**Reply by uto-lt** [reply]
Nice code! Thanks for your reply.
